server/audit_math/bravo.py

```diff
@@ -7,20 +7,21 @@
 targeted is being audited completely independently.
 """
 import math
-from decimal import Decimal, ROUND_CEILING
+from decimal import Decimal
 from collections import defaultdict
 import logging
 from typing import Dict, Tuple, Optional
 from scipy import stats
 
 from .sampler_contest import Contest
+from .ballot_polling_types import SampleSizeOption
 
 
-def get_expected_sample_sizes(
-    alpha: Decimal, contest: Contest, sample_results: Dict[str, int]
-) -> int:
+def get_expected_sample_size(
+    alpha: Decimal, contest: Contest, cumulative_sample_results: Dict[str, int]
+) -> SampleSizeOption:
     """
-    Returns the expected sample size for a BRAVO audit of <contest>
+    Returns the expected sample size or ASN (average sample number) for a BRAVO audit of <contest>
```

Comment:
Ah this is a linguistic ambiguity, technically "expected sample size" and "ASN" mean the same thing here.

Reply:
Reworded to make that clear: https://github.com/votingworks/arlo/pull/1619/commits/ceda9db32a577fbc5d8796220a3c7ed685d08fe5